### DISCUSSION TITLE
Next.js 如何在 i18n 场景下处理 404 页面 | liruifengv

### DISCUSSION THREAD

**Discussion by giscus** [liruifengv/liruifengv.com#29] [Announcements]
# Next.js 如何在 i18n 场景下处理 404 页面 | liruifengv

Next.js 使用 app route，国际化时无法自定义 404 页面。

https://liruifengv.com/posts/nextjs-404-in-i18n/

**Comment by Dreamer-Paul** [comment]
发现自定义页面之后，http 状态码变成 200 而不是 404 了，不知道大佬有没有发现（另外我记得之前评论过，为什么被吃掉了？
